modules/kubernetes/aks-core/variables.tf

```diff
@@ -242,6 +242,11 @@ variable "azure_policy_config" {
     ]
   }
 }
+variable "envoy_tls_policy_enabled" {
+  description = "An option to remove the gatekeeper mutation for tls settings"
+  type        = bool
+  default     = true
```

Comment:
Can we pls have this set to false instead?

Reply:
I don't think thats the best; I want users to make a conscious decision to disable the policy, i.e. have potentially insecure settings deployed in their cluster.

Reply:
Yeah, but for those customers that use azure policy, and not envoy, there will be mutations created anyway. In order to avoid that, we would have to add the variable in every architecture that doesn't use envoy. I prefer making sure that introducing new functionality and new variables doesn't require that we add those variables to existing infrastructures. Ideally, the infrastructures should be untouched by new functionality.